src/foam/nanos/u2/navigation/TopNavigation.js

```diff
@@ -97,6 +97,13 @@ foam.CLASS({
     }
   `,
 
+  messages: [
+    {
+      name: 'Greeting',
```

Comment:
Should this be GREETING (or WELCOME). But don't we name messages like constants?

Reply:
@kgrgreer If we're following suit to our previous messages then no, we use PascalCase.

Reply:
I looked through the project for how we use messages and we do it both ways in many different places.